README.md

```diff
@@ -1,36 +1,68 @@
-# Hello Microverse Project
+<a name="readme-top"></a>
 
-> This project has been built as part of the reviewers exercise or training.
-## Built With
+<div align="center">
 
-- HTML, CSS3
-- Github flow
-- VSCode
+  <h3><b>Microverse README Template</b></h3>
 
-## Authors
+</div>
 
-👤 **AJ193**
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+**[Hello Microverse]** is a simple html project containing one header.
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML and CSS
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Author1**
```

Comment:
- [ ] Please since you are the only contributor to this project you can singularize the `Authors` section to `Author` and replace `Author1` with your name. :smile: I also notice this author section is missing this back to top `<p align="right">(<a href="#readme-top">back to top</a>)</p>` which is very important.